spec/models/aroc_week_3_spec.rb

```diff
@@ -67,6 +69,7 @@
 
     # ------------------ Using ActiveRecord ----------------------
     # Solution goes here
+    # ???
```

Comment:
I wouldn't worry too much about this problem if you are stuck on it. The notion of getting "the third" of something in ActiveRecord/SQL doesn't make a ton of sense to me.